live.html

```diff
@@ -267,14 +267,18 @@ <h3 class="collection-header push-bottom">
   <div class="row">
     <div class="card-deck carousel" data-crds-carousel="mobile-scroll">
       <div class="feature-cards card-deck--expanded-layout">
-        {% assign recent_messages = site.messages | sort: 'date' | reverse | slice: 0,4 %} {% for message in recent_messages %}
+        {% assign recent_messages = site.messages | sort: 'date' | reverse | slice: 0,4 %} 
+        {% for message in recent_messages %}
+        {% for series in site.series %}

```

Comment:
Not sure that you need this extra loop. You could do the `assign` for a series and then set line `276` to `series.slug`.

Reply:
Looks like this extra loop may be messing up some stuff. Deploy preview looks like it has the same image for all 4 messages.

Reply:
Nah, they literally had the same image in contentful. I mixed it up

Reply:
I'm using the slug from the message to find the series it belongs to. I have to loop through the series and compare the message slug to the videos slug until I get a match. Unless I'm missing something?